misk/src/main/kotlin/misk/Actions.kt

```diff
@@ -21,7 +21,9 @@ fun KFunction<*>.asAction(
   val actualParameters = parameters.drop(1)
   val actionName = instanceParameter.type.classifier?.let {
     when (it) {
-      is KClass<*> -> it.simpleName
+      // Use the class name including all outer classes, but excluding the package (assuming a
+      // packages start with a lowercase character and classes start with an uppercase one).
+      is KClass<*> -> it.qualifiedName?.replace(Regex("^[a-z.]*"), "")
```

Comment:
Would `it.java.package?.name` work to get the package name?

Reply:
Oh, I like that a lot more. Will change up.